src/powersoftau_export_json.js

```diff
@@ -17,19 +17,20 @@
     along with snarkJS. If not, see <https://www.gnu.org/licenses/>.
 */
 
-import * as utils from "./powersoftau_utils.js";
+import * as potUtils from "./powersoftau_utils.js";
 import * as binFileUtils from "@iden3/binfileutils";
-import { stringifyBigIntsWithField } from "./misc.js";
+import { readG1, readG2 } from "./misc.js";
+import { utils as ffUtils} from "ffjavascript";
```

Comment:
```suggestion
import { stringifyBigIntsWithField, readG1, readG2 } from "./misc.js";
```
@kevjue Why did you change the stringifyBigIntsWithField here?